tests/CMakeLists.txt

```diff
@@ -252,10 +252,6 @@ endmacro()
 
 
 if(PYTHON_VERSION_MAJOR GREATER 2 AND SSG_PRODUCT_RHEL9)
-    ssg_refcheck_test("rhel9" "ccn_basic" "ccn")
-    ssg_refcheck_test("rhel9" "ccn_advanced" "ccn")
-    # This exclude can be removed once enable_authselect has a stigid
-    ssg_refcheck_test("rhel9" "stig" "stigid" "enable_authselect")
```

Comment:
Maybe it makes sense to start testing these references are in the final built data stream?

Reply:
Sorry for the post merge comment, @jan-cerny @Mab879

Reply:
For some products, there are CTest tests "verify-references-ssg-${PRODUCT}-ds.xml" and "missing-references-ssg-${PRODUCT}-ds.xml". I think we can extend them.

Reply:
see https://github.com/ComplianceAsCode/content/pull/11610